### ISSUE TITLE
Add prediction for cytoplasm and exosomes

### ISSUE THREAD

**Issue by mariaconcettavitale** [bgudenas/DeepLncRNA#6]
Hi,
How can I train the neural network to add prediction for cytoplasm and exosomes too?

**Comment by bgudenas** [comment]
Hi Conny,

DeepLncRNA was trained only on nuclear and cytosolic lncRNAs due to the small sample sizes for the other subcellular fractions within ENCODE at the time of publication. However, to extend the workflow to cytoplasmic and exosomal RNAs you can use the exact same workflow in the paper except you would label lncRNAs using the multi-class fraction (Nuclear, cyoplasmic, exosomal ...  then re-train the model. The new classification of lncRNAs would be based on differential expression (multiple groups --  likelihood ratio test ) analysis between the newly added RNA-seq samples (new fractions).
